starport/cmd/account_export.go

```diff
@@ -0,0 +1,65 @@
+package starportcmd
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+	"github.com/tendermint/starport/starport/pkg/cliquiz"
+	"github.com/tendermint/starport/starport/pkg/cosmosaccount"
+)
+
+func NewAccountExport() *cobra.Command {
+	c := &cobra.Command{
+		Use:   "export [name]",
+		Short: "Export an account",
+		Args:  cobra.ExactArgs(1),
+		RunE:  accountExportHandler,
+	}
+
+	c.Flags().AddFlagSet(flagSetKeyringBackend())
+	c.Flags().AddFlagSet(flagSetAccountImportExport())
+	c.Flags().String(flagPath, "", "path to export private key. default: ./key_[name]")
+
+	return c
+}
+
+func accountExportHandler(cmd *cobra.Command, args []string) error {
+	var (
+		name       = args[0]
+		path, _    = cmd.Flags().GetString(flagPath)
+		passphrase = getPassphrase(cmd)
+	)
+
+	if passphrase == "" && !getIsNonInteractive(cmd) {
+		if err := cliquiz.Ask(cliquiz.NewQuestion("Passphrase", &passphrase, cliquiz.HideAnswer())); err != nil {
```

Comment:
We should have a "Confirm passphrase" prompt

Reply:
97a50586